README.md

```diff
@@ -1,195 +1,24 @@
-
-## Configuration
-
-By default, you can configure pouchdb-server using a `config.json` file, which is
-typically expected at the root of wherever you run pouchdb-server, but may be specified with the `--config` option.
-
-Below are some examples of `config.json` options:
-
-### log.level
-
-To change the log output level, you can create a `config.json` file containing e.g.:
-
-```js
-{
-  "log": {
-    "level": "none"
-  }
-}
-
-```
-
-The available values are `debug`, `info`, `warning`, `error`, and `none`. The default
-is `info`.
-
-### log.file
-
-To choose the file where logs are written, you can create a `config.json` file containing e.g.:
-
-```js
-{
-  "log": {
-    "file": "/path/to/log.txt"
-  }
-}
-
-```
-
-By default, logs are written to `./log.txt`.
-
-## Automatic port configuration
-
-Due to conventions set by Heroku and others, if you have a `PORT` environment variable,
-`pouchdb-server` will pick up on that and use it instead of `5984` as the default.
-
-```bash
-export PORT=3000
-pouchdb-server # will run on port 3000
-```
-
-## Testing
-
-One of the primary benefits of **pouchdb-server** is the ability to run PouchDB's Node test suite against itself. To do that, you can simply,
-
-```bash
-$ npm run test-pouchdb
-```
-
-Whatever args you provide as `SERVER_ARGS` will be passed to `pouchdb-server` itself:
-
-```bash
-$ SERVER_ARGS='--in-memory' npm run test-pouchdb
-```
-
-Or to test in Firefox (IndexedDB):
-
-```bash
-$ CLIENT=selenium:firefox npm run test-pouchdb
-```
-
-Or to test in PhantomJS (WebSQL):
-
-```bash
-$ CLIENT=selenium:phantomjs ES5_SHIM=true npm run test-pouchdb
-```
-
-Additionally, we've started porting CouchDB's JavaScript test harness to
-[a simple Node module](https://github.com/nick-thompson/couchdb-harness), which can be run against PouchDB via **pouchdb-server**.
-
-```bash
-$ npm run test-couchdb
-```
 
 ## Contributing
 
-Want to help me make this thing awesome? Great! Here's how you should get started.
-
-1. First, make sure that the bugfix or feature you're looking to implement isn't better fit for [express-pouchdb](https://github.com/nick-thompson/express-pouchdb).
-2. PouchDB is still developing rapidly. If you need bleeding egde versions, you should first read how to [set up express-pouchdb for local development](https://github.com/nick-thompson/express-pouchdb#contributing). (Make sure that, afterwards, you `npm link` express-pouchdb).
-3. Go ahead and fork **pouchdb-server**, clone it to your machine.
-4. Now you'll want to, from the root of **pouchdb-server**, `npm link express-pouchdb`.
-5. `npm install` the rest of the dependencies.
-
-Please make your changes on a separate branch whose name reflects your changes, push them to your fork, and open a pull request!
-
-For commit message style guidelines, please refer to [PouchDB CONTRIBUTING.md](https://github.com/pouchdb/pouchdb/blob/master/CONTRIBUTING.md).
+See the CONTRIBUTING.md file for how to get involved
```

Comment:
maybe link the CONTRIBUTING.md file? `[CONTRIBUTING.md](CONTRIBUTING.md)` should work I think